distribution/docs/src/main/resources/_contents/_sources/wfs-10-source-contents.adoc

```diff
@@ -0,0 +1,42 @@
+[[_wfs_v100_source]]
+==== WFS v1.0.0 Source
+
+The WFS Source allows for requests for geographical features across the web using platform-independent calls.
+
+A Web Feature Service (WFS) source is an implementation of the `FederatedSource` interface provided by the ${branding} Framework.
+
+Use the WFS Source if querying a WFS version 1.0.0 compliant service.
+
+===== Installing the WFS v1.0.0 Source
+
+The WFS v1.0.0 Source is is not installed by default with a standard installation in the ${ddf-spatial} application.
+
+===== Configuring the WFS v1.0.0 Source
+
+* Navigate to the ${admin-console}.
+* Navigate to the ${ddf-spatial} application.
+*
```

Comment:
Should this asterisk be here?
![image](https://cloud.githubusercontent.com/assets/8041246/18294578/29064972-744f-11e6-980a-ab03c153ef7c.png)